apps/test/unit/templates/sectionsRefresh/QuickAssignTableTest.jsx

```diff
@@ -58,6 +59,21 @@ describe('QuickAssignTable', () => {
     expect(updateSpy).to.have.been.called;
   });
 
+  it('correctly falls back when a course has no recommended version', () => {
+    const updateSpy = sinon.spy();
+    const wrapper = setUpMount({
+      updateCourse: updateSpy,
+      courseOfferings: noRecommendedVersionsOfferings,
+    });
+
+    const radio = wrapper.find('input');
```

Comment:
Thanks for adding this test! The only thing I can think of would be to add one more stable version of a different year to ensure this is grabbing the most recent one.